msg/Arrays.msg

```diff
@@ -0,0 +1,34 @@
+# Arrays of different types
+bool[3] bool_values
+byte[3] byte_values
+char[3] char_values
+float32[3] float32_values
+float64[3] float64_values
+int8[3] int8_values
+uint8[3] uint8_values
+int16[3] int16_values
+uint16[3] uint16_values
+int32[3] int32_values
+uint32[3] uint32_values
+int64[3] int64_values
+uint64[3] uint64_values
+string[3] string_values
+BasicTypes[3] basic_types_values
+Constants[3] constants_values
+Defaults[3] defaults_values
+bool[3] bool_values_default [true, false, true]
+byte[3] byte_values_default [1, 0, 100]
+char[3] char_values_default [1, 0, 200]
+float32[3] float32_values_default [1.125, 0.0, -1.125]
+float64[3] float64_values_default [3.1415, 0.0, -3.1415]
+int8[3] int8_values_default [10, 0, -10]
+uint8[3] uint8_values_default [1, 0, 200]
+int16[3] int16_values_default [3000, 0, -3000]
+uint16[3] uint16_values_default [1, 0, 4000]
+int32[3] int32_values_default [500000, 0, -500000]
+uint32[3] uint32_values_default [1, 0, 6000000]
+int64[3] int64_values_default [70000000, 0, -70000000]
+uint64[3] uint64_values_default [1, 0, 800000000]
+string[3] string_values_default ["Foo", "", " "]
```

Comment:
I would suggest to also use the default values from the current fixtures here.

Reply:
90395df